Dockerfile

```diff
@@ -109,7 +109,7 @@ RUN --mount=type=cache,target=/app/.meteor/local/ meteor build --allow-superuser
 
 # Install server dependencies
 WORKDIR /built_app/bundle/programs/server
-RUN --mount=type=cache,target=/root/.npm meteor --allow-superuser npm install --production --omit=optional
+RUN --mount=type=cache,target=/root/.npm meteor npm --allow-superuser install --production --omit=optional
```

Comment:
CI revealed that the new version of the `meteor` entrypoint seems to require that if you're chaining to a subtool like `npm` or `npx`, that argument be precisely first in argv, so the `--allow-superuser` needed to move back, even though it's weird because the arg isn't really intended for the `npm` command.  🤷🏻